doc/sphinx/src/models.rst

```diff
@@ -214,40 +230,66 @@ The user should note that this implies that :math:`e=0` at the reference
 temperature, :math:`T_0`. Given this simple relationship, the user should
 treat the temperature from this EOS as only a rough estimate.
 
-The Grunesien parameter is given by
+First defining the compression as
+
+.. math::
+
+    \eta = 1 - \frac{\rho_0}{\rho},
+
+where :math:`\rho_0` is the reference density, the Grunesien parameter is given
+by
 
 .. math::
 
     \Gamma(\rho) =
       \begin{cases}
         \Gamma_0                                          & \rho < \rho_0 \\
-        \Gamma_0 \frac{\rho_0}{\rho} 
-           + b(1 - \frac{\rho_0}{\rho})                   & \rho >= \rho_0
+        \Gamma_0 (1 - \eta) + b\eta                       & \rho \geq \rho_0
       \end{cases}
 
-and when the unitless user parameter :math:`b=0`, this ensures the the Gruneisen
+When the unitless user parameter :math:`b=0`, this ensures the the Gruneisen
 parameter is of a form where :math:`\rho\Gamma =` constant in compression.
```

Comment:
Given that compression is now a variable, this phrasing has become confusing I think. Maybe "in the so-called compression limit, when when eta < 0"

Reply:
I think including rho Gamma = constant is good, I just want to reference what "in compression" means here.

Reply:
Ah gotcha. That makes sense! I'll change that